src/core/src/op/strided_slice.cpp

```diff
@@ -159,26 +142,15 @@ void op::v1::StridedSlice::validate_and_infer_types() {
     set_input_is_relevant_to_shape(2);
     set_input_is_relevant_to_shape(3);
 
-    auto begin_const = get_constant_from_source(input_value(1));
-    auto end_const = get_constant_from_source(input_value(2));
-    auto strides = get_constant_from_source(input_value(3));
-
-    if (begin_const && end_const && strides) {
-        set_output_type(0,
-                        get_input_element_type(0),
-                        infer_slice_shape(this,
```

Comment:
Do we use this method anywhere else? If not, can we remove it?

Reply:
removed!

Reply:
Reverted,  infer_slice_shape() is used by external project like VPU compiler,